draft-ietf-trans-rfc6962-bis.md

```diff
@@ -757,39 +757,60 @@ Final STH:
 [JSON.Metadata] is an example of a metadata format which includes the above
 elements.
 
-## Accepting Submissions
-
-To ensure that logged certificates and precertificates are attributable to a
-known trust anchor, and to set clear expectations for what monitors would find
-in a log, and to avoid being overloaded by invalid submissions, the log MUST NOT
-accept any submission until it has verified that the submitted certificate or
-precertificate chains to an accepted trust anchor.
-
-The log MUST NOT use other sources of intermediate CA certificates to attempt
-certification path construction; instead, it MUST only use the intermediate CA
-certificates provided in the submission, in the order provided.
-
-Logs SHOULD accept certificates and precertificates that are fully valid
-according to RFC 5280 [RFC5280] verification rules and are submitted with such
-a chain. (A log may decide, for example, to temporarily reject valid submissions
-to protect itself against denial-of-service attacks).
-
-Logs MAY accept certificates and precertificates that have expired, are not yet
-valid, have been revoked, or are otherwise not fully valid according to RFC
-5280 verification rules in order to accommodate quirks of CA
-certificate-issuing software.
-However, logs MUST reject submissions without a valid signature chain to an
-accepted trust anchor. Logs MUST also reject precertificates that do not conform
-to the requirements in {{precertificates}}.
+## Evaluating Submissions
 
-Logs SHOULD limit the length of chain they will accept. The maximum chain length
-is one of the log's parameters (see {{log_parameters}}).
+A log determines whether to accept or reject a submission by evaluating it
+against the minimum acceptance criteria (see {{minimum_criteria}}) and against
+the log's discretionary acceptance criteria (see {{discretionary_criteria}}).
+
+If the acceptance criteria are met, the log SHOULD accept the submission. (A log
+may decide, for example, to temporarily reject acceptable submissions to protect
+itself against denial-of-service attacks).
+
+### Minimum Acceptance Criteria {#minimum_criteria}
 
 The log SHALL allow retrieval of its list of accepted trust anchors (see
```

Comment:
Nit: It seems like this section belongs at the end / outside the minimum acceptance criteria section? It seems out of place at the beginning of this  section.

Reply:
Good point.  Addressed by https://github.com/robstradling/certificate-transparency-rfcs/commit/e3f549fbf5149092a15c24808e1352ed512b0530